astroquery/astorbdb/core.py

```diff
@@ -0,0 +1,720 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import astropy.units as u
+from astropy.time import Time
+import json
+from collections import OrderedDict
+
+from ..query import BaseQuery
+from ..utils import async_to_sync
+from . import conf
+
+__all__ = ['AstInfo', 'AstInfoClass']
+
+
+@async_to_sync
+class AstInfoClass(BaseQuery):
+
+    """
+    A class for querying Lowell Observatory's `astorbDB
+    <https://asteroid.lowell.edu/>`_ service.
+    """
+    # Not all the methods below are necessary but these cover most of the common
+    # cases, new methods may be added if necessary, follow the guidelines at
+    # <http://astroquery.readthedocs.io/en/latest/api.html>
+
+    URL = conf.server
+    TIMEOUT = conf.timeout
+
+    # internal flag whether to return the raw reponse
+    _return_raw = False
+
+    # actual query uri
+    _uri = None
+
+    def designations_async(self, object_name, *,
+                           get_raw_response=False,
+                           get_uri=False,
```

Comment:
I don't think this is needed either.  Maybe you were following the jplhorizons module? (which is ancient frankencode)

Reply:
Similarly, remove the related if statement, and edit the other methods.